jekyll/_cci2/orb-concepts.md

```diff
@@ -7,203 +7,137 @@ categories: [getting-started]
 order: 1
 ---
 
-CircleCI orbs are shareable packages of configuration elements, including jobs, commands, and executors. CircleCI provides certified orbs, along with 3rd-party orbs authored by CircleCI partners. It is best practice to first evaluate whether any of these existing orbs will help you in your configuration workflow. Refer to the CircleCI Orbs Registry for the complete list of certified orbs.
+[CircleCI orbs](https://circleci.com/orbs/) are shareable packages of configuration elements, including jobs, commands, and executors. Orbs make writing and customizing CircleCI config simple. The reusable configuration elements used in orbs are explained fully in the [Reusable Configuration Reference]({{site.baseurl}}/2.0/reusing-config/).
 
-Before using orbs, you should first familiarize yourself with some basic core concepts of orbs and how they are structured and operate. Gaining a basic understanding of these core concepts will enable you to leverage orbs and use them easily in your own environments.
+## Reusable Configuration
 
-### Certified vs. 3rd-Party Orbs
+CircleCI's [Reusable Configuration]({{site.baseurl}}/2.0/reusing-config/) features allow you to define parametrizable configuration elements and re-use those elements throughout a project config file. It is recommended you become familiar with the full [Configuration Reference]({{site.baseurl}}/2.0/configuration-reference/) features before moving on to the [Reusable Configuration Reference]({{site.baseurl}}/2.0/reusing-config/).
 
-CircleCI has available a number of individual orbs that have been tested and certified to work with the platform. These orbs will be treated as part of the platform; all other orbs are considered 3rd-party orbs. Note: The Admin of your org must opt-in to 3rd-party uncertified orb usage on the Settings > Security page for your org.
+### Commands
+Commands contain one or more [steps]() in which [parameters]() can be used to modify behavior. Commands are the logic of orbs and are responsible for executing steps such as [checking out code](https://circleci.com/docs/2.0/configuration-reference/#checkout), or running shell code, for example running BASH or running CLI tools.
 
-All orbs are open, meaning that anyone can use them and see their source.
+For more information see our guide to [Authoring Reusable Commands]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-commands).
 
-### Design Methodology
+As an example, the AWS S3 orb includes a _command_ to copy a file or object to a new location. If your AWS authentication details are pre-stored as environment variables the syntax to use this command in your config is simply:
 
-Before using orbs, you may find it helpful to understand the various design decisions and methodologies that were used when these orbs were designed. Orbs were designed with the following considerations:
-
-* Orbs are transparent - If you can execute an orb, you and anyone else can view the source of that orb.
```

Comment:
Is this list no longer correct, @KyleTryon?

Reply:
The Design Methodology section? This seemed to cover some redundant topics, such as semver and immutability (although maybe a quick note on immutability do that). Same with the piece on organization admins, that is covered elsewhere and should probably be touched on again in the development page but seemed redundant here.

will send over a small change for the immutability piece

Reply:
Expanded on the orb versions section, I think this addresses this.

Reply:
Brilliant, thanks!